### PR TITLE
Fixes powergrids not working in away missions

### PR DESCRIPTION
**Pull request by Kyep**
🆑 Kyep
fix: Powergrids in away missions now work. This addresses doors not being powered, APCs not charging, etc.
/🆑

### CODE REVIEW

File: code/modules/awaymissions/zlevel.dm [hunk 1]
```diff
@@ -58,7 +58,7 @@ var/global/list/potentialRandomZlevels = generateMapList(filename = "config/away
 			late_setup_level(block(locate(1, 1, zlev), locate(world.maxx, world.maxy, zlev)))
 			space_manager.remove_dirt(zlev)
```
Comment (Crazylemon64) [inline code/modules/awaymissions/zlevel.dm]: This *should* be handled by the `remove_dirt` hooks - there's one in particular that takes care of powernets. Is something amiss with that one?

Comment (Crazylemon64) [inline code/modules/awaymissions/zlevel.dm]: https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/space_management/space_level.dm#L179-L187

Comment (Kyep) [inline code/modules/awaymissions/zlevel.dm]: Yes. It doesn't work. And https://github.com/ParadiseSS13/Paradise/blob/c973875b95ed4a58415cc3e341d0b8952f6a933b/code/modules/space_management/zlevel_manager.dm#L77 does not seem to reference either hooks, or powernets. This PR makes it work.

**Comment by Kyep** [comment]
Closing, as CL said he'd prefer to fix this himself by messing with the datums.

**Pull request closed without merging**